src/applescript/dialog.js

```diff
@@ -0,0 +1,42 @@
+const exec = require('./exec.js');
+
+const files = require('./files.js');
+
+const url =
+  'https://user-images.githubusercontent.com/1034890/209779279-7b23e061-c5af-4129-9d51-1e674a0818e8.jpg';
+
+async function showDialog(
+  title,
+  text,
+  buttons,
+  givingUpAfter = 30,
+  showIcon = false,
+) {
+
+  const path = files(url)
+  const SCRIPT = ` 
+--set theImage to alias of thePath as string
+set theImage to (path to temporary items as text) & "focusbear.png"
```

Comment:
we should be able to reference an absolute path. The swift code will put the image there. /Users/<USERNAME>/Library/Application Support/com.focusbear.latenomore/icon.png